libraries/Microsoft.Bot.Builder/Teams/TeamsInfo.cs

```diff
@@ -338,6 +338,140 @@ public static async Task<MeetingNotificationResponse> SendMeetingNotificationAsy
             }
         }
 
+        /// <summary>
+        /// Sends a message to the provided list of Teams members.
+        /// </summary>
+        /// <param name="turnContext"> Turn context. </param>
+        /// <param name="activity"> The activity to send. </param>
+        /// <param name="teamsMembers"> The list of members. </param>
+        /// <param name="tenantId"> The tenant ID. </param>
+        /// <param name="cancellationToken"> The cancellation token. </param>
+        /// <returns> The operation Id. </returns>
+        public static async Task<string> SendMessageToListOfUsersAsync(ITurnContext turnContext, IActivity activity, List<object> teamsMembers, string tenantId, CancellationToken cancellationToken = default)
```

Comment:
Is there a reason why we're using object and not specifying a concrete type for list of members?

Reply:
We created a _TeamMember_ model with a string Id property to handle this.